frontend/src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from "react";
+import { Container, Wrapper } from "./HeaderStyle";
+import { Link } from "react-router-dom";
+import { Navbar } from "../draft/NavBar/Navbar";
+
+const Header: React.FC = ({}) => {
+  const [toggleMenu, setToggleMenu] = useState(false);
+  const [loggedIn, setLoggedIn] = useState(true);
+  const [screenWidth, setScreenWidth] = useState(window.innerWidth);
```

Comment:
> const [screenWidth, setScreenWidth] = useState(window.innerWidth);

you can use [react-responsive](https://www.npmjs.com/package/react-responsive) instead